examples/example-rust-vm/Cargo.toml

```diff
@@ -1,8 +1,13 @@
+# EVMC: Ethereum Client-VM Connector API.
+# Copyright 2019 The EVMC Authors.
+# Licensed under the Apache License, Version 2.0.
+
 [package]
 name = "example-rust-vm"
-version = "0.1.0"
+version = "7.4.0-alpha.0"
 authors = ["Alex Beregszaszi <alex@rtfs.hu>", "Jake Lang <jak3lang@gmail.com>"]
 edition = "2018"
+publish = false
```

Comment:
What does `publish` do?

Reply:
It wont allow one to erroneously run `cargo publish` pushing it to crates.io.